### PR TITLE
Convert `packages/grafana-ui/src/components/Logs/LogMessageAnsi.test.tsx` to RTL

### PR DESCRIPTION
**Pull request by xrisaD**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by xrisaD** [comment]
Hi @ashharrison90, 
could you help me on that one? ⛑️

File: packages/grafana-ui/src/components/Logs/LogMessageAnsi.test.tsx [hunk 1]
```diff
@@ -7,44 +7,40 @@ import { UnThemedLogMessageAnsi as LogMessageAnsi } from './LogMessageAnsi';
...
-    expect(wrapper.find('span')).toHaveLength(1);
-    expect(wrapper.find('span').first().prop('style')).toMatchObject(
-      expect.objectContaining({
-        color: expect.any(String),
-      })
-    );
-    expect(wrapper.find('span').first().text()).toBe('ipsum');
+    render(<LogMessageAnsi value={value} theme={createTheme()} />);
+
+    //expect(wrapper.find('span')).toHaveLength(1);
```
Comment (ashharrison90) [inline packages/grafana-ui/src/components/Logs/LogMessageAnsi.test.tsx:19]: this can be removed

**Review by ashharrison90 — COMMENTED** [review]
looks good - just a stray comment to remove i think 😄

File: packages/grafana-ui/src/components/Logs/LogMessageAnsi.test.tsx (hunk 1 above)
Comment (xrisaD) [reply packages/grafana-ui/src/components/Logs/LogMessageAnsi.test.tsx:19]: oops, I will remove it sorry 😄

**Review by ashharrison90 — APPROVED** [review]
looks great! thanks again @xrisaD

**Pull request merged**